docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 web:
   build: .
-  command: mix phoenix.server
+  command: service nginx start && mix phoenix.server
```

Comment:
Also @barry no it doesn't handle failures on nginx as that process is not
attached. But If APR doesn't exit cleanly it will restart the container.
Also as in @joey's case... Multiple CMD statements are not supported which
is why the community seems to recommend using supervisor / monit as well

On Sep 8, 2016 11:58, "Barry Hoggard" <notifications@github.com> wrote:

> In docker-compose.yml
> <https://github.com/artsy/APR/pull/26#discussion_r78035614>:
>
> > @@ -1,11 +1,11 @@
> >  web:
> >    build: .
> > -  command: mix phoenix.server
> > +  command: service nginx start && mix phoenix.server
>
> You can remove command if the one in the Dockerfile is OK to use with
> docker-compose. @joeyAghion <https://github.com/joeyAghion> Gravity uses
> a common.yml file:
>
> https://github.com/artsy/gravity/blob/master/docker-compose.yml
> https://github.com/artsy/gravity/blob/master/common.yml
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/artsy/APR/pull/26/files/88919632c1a40b79466607aa874e3b2e2c58f75c#r78035614>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA847a5bMgcJfs3EOP6ACza0Z_W7IhsOks5qoDCngaJpZM4J3QeA>
> .
>